pxe/PXELinux_default.erb

```diff
@@ -17,6 +17,13 @@ LABEL local
      MENU DEFAULT
      LOCALBOOT 0
 
+LABEL discovery
+     MENU LABEL (discovery)
+     MENU DEFAULT
+     KERNEL boot/fdi-image/vmlinuz0
+     APPEND initrd=boot/fdi-image/initrd0.img rootflags=loop root=live:/fdi.iso rootfstype=auto ro rd.live.image acpi=force rd.luks=0 rd.md=0 rd.dm=0 rd.lvm=0 rd.bootif=0 rd.neednet=0 nomodeset proxy.url=https://FOREMAN_INSTANCE proxy.type=foreman
```

Comment:
I tired and actually it is complicated, I'd rather have a first cut as a manual edit.

https://github.com/theforeman/foreman/pull/1827